Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.py

```diff
@@ -685,6 +685,27 @@ def _get_attachment_mime(self, message_id, attachment_id, overwrite_rate_limit_r
 
         return mime_content
 
+    @staticmethod
+    # -*- coding: utf-8 -*-
+    def _is_english(s):
```

Comment:
What do you 'other chars'?
It should return False just in case the string contains characters from some other alphabet,
but in case of Latin alphabet and some special characters (like #@!$%^&&*) should return True

Reply:
let's change it to `_only_ascii`